ProcessMaker/modeler
# Add base bpmnlint plugin and Inclusive Gateway validation

Fixes #268.

The PR adds custom BPMNLint validation. It adds validation rules for the following gateway constrains:
* A **Gateway** with a `gatewayDirection` of `converging` MUST have multiple _incoming_ **Sequence Flows**, but MUST NOT have multiple _outgoing_ **Sequence Flows**.
* A **Gateway** with a `gatewayDirection` of `diverging` MUST have multiple _outgoing_ **Sequence Flows**, but MUST NOT have multiple _incoming_ **Sequence Flows**.

**To test:**
Drag a gateway onto the grid, and check that the above errors correctly appear when validating the diagram.
